requirements/test-ci-default.txt

```diff
@@ -12,7 +12,7 @@
 -r extras/thread.txt
 -r extras/elasticsearch.txt
 -r extras/couchdb.txt
-#-r extras/couchbase.txt
+-r extras/couchbase.txt
```

Comment:
Why this is uncommented?

Reply:
Because it looks like a [typo](https://github.com/celery/celery/commit/8a92b7128bc921d4332fe01486accc243115aba8) in previous commit. If there is a reason to leave it commented, I'll fix it.

Reply:
this can be ignored for this PR :D